apps/passport-server/test/zupass.spec.ts

```diff
@@ -33,33 +54,95 @@ describe("zupass functionality", function () {
   this.timeout(15_000);
 
   let application: PCDpass;
+  let emailAPI: IEmailAPI;
+  let pretixMocker: ZuzaluPretixDataMocker;
+  let pretixService: ZuzaluPretixSyncService;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  let devconnectPretixSyncService: DevconnectPretixSyncService;
+  let mocker: DevconnectPretixDataMocker;
+  let db: Pool;
+  let server: SetupServer;
+
   let residentUser: User | undefined;
+  let residentIdentity: Identity | undefined;
   let visitorUser: User | undefined;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  let visitorIdentity: Identity | undefined;
```

Comment:
If this isn't used, we can just get rid of it. It adds noise to the test by having a value assigned to it which we never do anything with.

Reply:
I promise I'll get rid of these ugly comments this week! In some sense this PR is a WIP - i'm in the process of merging `devconnect` and `zupass` spec